_posts/2019-05-15-cockpit-194.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 194
+author: lis
+date: 2019-05-15
+tags: cockpit linux
+slug: cockpit-194
+category: release
+summary: Firewall page improvements and a new switch
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 194.
+
+### Firewall: Add services to a specific zone
+
+Services can now be added to and removed from specific zones.
+
+![Adding services to zones](/images/firewall-add-services-to-zones.png)
+
+Furthermore, the firewall page will now give limited feedback in case of
+errors.
```

Comment:
This sounds a bit odd -- how about "Dialogs on the Firewall page now show error messages"?